src/test/java/apicurito/tests/steps/MainPageSteps.java

```diff
@@ -149,8 +126,138 @@ public void selectDataType(String dataTypeName) {
         MainPageUtils.getDataTypeWithName(dataTypeName).click();
     }
 
-    @Then("^delete API \"([^\"]*)\"$")
-    public void deleteAPI(String file) {
-        new File(file).delete();
+    @And("^create basic security scheme with values$")
```

Comment:
I created these 3 methods on purpose in this way. 
The first reason is readability of the code. One method with 100 rows is less readable than 3 methods with approx 30 rows. 
The second reason is the size of the data table. If I use one table for all schemes, the table will have 8 rows. e.g.
| SchemeType | SchemeName | Scheme description | KeyLocation | Header | Flow | AuthUrl | TokenUrl |

And some groups of rows are related just to one scheme. So If I create a basic scheme without description it will looks like:
| Basic | MyName | | | | | | |

Api key : 
| Api Key | MyName |  | KeyLocation | Header | | | 

In my opinion less readable than 3 methods each with the proper parameters.